.circleci/config.yml

```diff
@@ -345,6 +341,37 @@ jobs:
       - run:
           name: Push images
           command: docker-compose -f docker-compose.ci.yml push
+      - run:
+          name: Archive docker images
+          command: |
+            docker tag ${AWS_REGISTRY}/digideps/client:${VERSION} client:latest
+            docker save -o docker-images.tar client:latest
+      - persist_to_workspace:
+          root: .
+          paths:
+          - ./docker-images.tar
+
+  client-unit-test:
+    docker:
+      - image: circleci/python:2.7
```

Comment:
Is there a reason we can't use a later version of Python? Or is 2.7 still in support?